src/Common/test-runtime/project.json

```diff
@@ -2,12 +2,15 @@
    "dependencies": {
       "xunit.console.netcore": "1.0.3-prerelease-00921-01",
       "Microsoft.DotNet.BuildTools.TestSuite": "1.0.1-prerelease-01001-04",
-      "Microsoft.xunit.netcore.extensions": "1.0.1-prerelease-01001-04"
+      "Microsoft.xunit.netcore.extensions": "1.0.1-prerelease-01001-04",
+      "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24728-02",
```

Comment:
Should we instead keep this change local only since Wes will have to remove it and the actual tests aren't running as part of CI?